examples/src/main/scala/sqlest/examples/Extractor.scala

```diff
@@ -21,6 +21,25 @@ object ExtractorExamples extends App with DatabaseExample {
     name = FruitTable.name,
     juiciness = FruitTable.juiciness
   )
+
+  // extractors can be used in the declaration of other extractors
+  lazy val smoothyExtractor = extract[Smoothy](
+    description = SmoothyTable.description,
+    fruits = fruitExtractor.asList
+  ).groupBy(SmoothyTable.id)
+
+  // inner joins can be used as follows
+  val smoothies =
+    select
+      .from(
```

Comment:
Sorry I missed this the first time.
You can now write the inner joins separately to the from clauses so it looks more like regular sql:
```scala
  .from(SmoothyTable)
  .innerJoin(IngredientsTable).on(SmoothyTable.id === IngredientsTable.smoothyId)
  .innerJoin(FruitTable).on(IngredientsTable.fruitId === FruitTable.id)
```